### PR TITLE
add cors check

### PR DESCRIPTION
**Pull request by shovanmaity**
## Proposed changes

Summarize your changes here to communicate with the maintainers and make sure to put the link of that issue

## Types of changes

What types of changes does your code introduce to Litmus? Put an `x` in the boxes that apply
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation Update (if none of the other choices applies)

## Checklist

Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code.
- [ ] I have read the [CONTRIBUTING](https://github.com/litmuschaos/litmus/blob/master/CONTRIBUTING.md) doc
- [ ] I have signed the commit for DCO to be passed.
- [ ] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works (if appropriate)
- [ ] I have added necessary documentation (if appropriate)

## Dependency
- Please add the links to the dependent PR need to be merged before this (if any).

### CODE REVIEW

**Review by imrajdas — APPROVED** [review]

File: chaoscenter/authentication/api/main.go [hunk 1]
```diff
@@ -29,22 +29,22 @@ import (
...
-	DbServer      string `required:"true" split_words:"true"`
-	DbUser        string `required:"true" split_words:"true"`
-	DbPassword    string `required:"true" split_words:"true"`
+	JwtSecret      string   `required:"true" split_words:"true"`
+	AdminUsername  string   `required:"true" split_words:"true"`
+	AdminPassword  string   `required:"true" split_words:"true"`
+	DbServer       string   `required:"true" split_words:"true"`
+	DbUser         string   `required:"true" split_words:"true"`
+	DbPassword     string   `required:"true" split_words:"true"`
+	AllowedOrigins []string `split_words:"true" default:"litmus.io?,localhost:([0-9]+)?"`
```
Comment (Saranya-jena) [inline chaoscenter/authentication/api/main.go:38]: i think it should be `litmuschaos.io` . @Jonsy13 @imrajdas  can you confirm?

File: chaoscenter/graphql/server/utils/variables.go [hunk 2]
```diff
@@ -5,45 +5,46 @@ var (
...
+	SelfInfraTolerations        string   `split_words:"true"`
+	HttpPort                    string   `split_words:"true" default:"8080"`
+	RpcPort                     string   `split_words:"true" default:"8000"`
+	InfraCompatibleVersions     string   `required:"true" split_words:"true"`
+	DefaultHubGitURL            string   `required:"true" default:"https://github.com/litmuschaos/chaos-charts"`
+	DefaultHubBranchName        string   `required:"true" split_words:"true"`
+	CustomChaosHubPath          string   `split_words:"true" default:"/tmp/"`
+	DefaultChaosHubPath         string   `split_words:"true" default:"/tmp/default/"`
+	EnableGQLIntrospection      string   `split_words:"true" default:"false"`
+	AllowedOrigins              []string `split_words:"true" default:"litmus.io?,localhost:([0-9]+)?"`
```
Comment (Saranya-jena) [inline chaoscenter/graphql/server/utils/variables.go:47]: same

**Review by SarthakJain26 — APPROVED** [review]

**Review by amityt — APPROVED** [review]

File: chaoscenter/authentication/api/main.go (hunk 1 above)
Comment (shovanmaity) [reply chaoscenter/authentication/api/main.go]: Changed to litmuschaos.io

**Review by Jonsy13 — APPROVED** [review]

File: chaoscenter/graphql/server/utils/variables.go [hunk 3]
```diff
@@ -5,45 +5,46 @@ var (
...
+	SelfInfraTolerations        string   `split_words:"true"`
+	HttpPort                    string   `split_words:"true" default:"8080"`
+	RpcPort                     string   `split_words:"true" default:"8000"`
+	InfraCompatibleVersions     string   `required:"true" split_words:"true"`
+	DefaultHubGitURL            string   `required:"true" default:"https://github.com/litmuschaos/chaos-charts"`
+	DefaultHubBranchName        string   `required:"true" split_words:"true"`
+	CustomChaosHubPath          string   `split_words:"true" default:"/tmp/"`
+	DefaultChaosHubPath         string   `split_words:"true" default:"/tmp/default/"`
+	EnableGQLIntrospection      string   `split_words:"true" default:"false"`
+	AllowedOrigins              []string `split_words:"true" default:"litmuschaos.io?,localhost:([0-9]+)?"`
```
Comment (Jonsy13) [inline chaoscenter/graphql/server/utils/variables.go:47]: Should we add github.com here as well? Considering ChaosHub & GitOps via GitHub?

Comment (shovanmaity) [reply chaoscenter/graphql/server/utils/variables.go:47]: No, This is hosted portal url. User need to configure it during deployment time.

Comment (Jonsy13) [reply chaoscenter/graphql/server/utils/variables.go:47]: Okay if tested, then it's fine!

**Pull request merged**